<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardWindow</class>
 <widget class="QDialog" name="BoardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шахматы :: Доска</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="boardWidget" native="true">
     <property name="styleSheet">
      <string>background-color: white;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string>background-color: lightgreen; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Запись в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="styleSheet">
        <string>background-color: lightcoral; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
